paasta_tools/generate_authenticating_services.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A simple script to enumerate all services partecipating in authenticated
```

Comment:
oops just noticed this typo :) `partecipating`

Reply:
itanglish strikes again